turborepo-tests/integration/tests/interactive.t

```diff
@@ -0,0 +1,15 @@
+Setup
+  $ . ${TESTDIR}/../../helpers/setup_integration_test.sh
+  $ . ${TESTDIR}/../../helpers/replace_turbo_json.sh $(pwd) "interactive.json"
+Verify we error on interactive task that hasn't been marked as cache: false
```

Comment:
not needed for this PR, but are we still planning to:
- assume `cache:false` when it isn't specified manually?
- allow `cache:true` if it's set explicitly?